package.json

```diff
@@ -44,16 +44,11 @@
     "@types/meow": "^5.0.0",
     "@types/node": "^14.14.32",
     "@types/prettier": "^1.18.0",
-    "colors": "^1.3.3",
     "contentful": "^8.1.9",
-    "coveralls": "^3.0.5",
-    "cross-env": "^5.2.0",
     "cz-conventional-changelog": "^3.0.2",
     "husky": "^3.0.2",
     "jest": "^26.0.0",
-    "jest-config": "^26.0.0",
     "lint-staged": "^9.2.1",
-    "lodash.camelcase": "^4.3.0",
```

Comment:
`lodash` is a dependency (line 33), so no need to import this. I'd ideally prefer to import specific packages because we only use `camelcase` and `upperfirst`, but some of our dependencies like jsdom need the full library so there's no point spending time on that